EndlessClient/Controllers/InventoryController.cs

```diff
@@ -230,7 +234,10 @@ public void DropItem(EIFRecord itemData, InventoryItem inventoryItem)
             }
             else if (validationResult == ItemDropResult.TooFar)
             {
+                var message = "Out of drop range.";
```

Comment:
Inject `ILocalizedStringFinder` and use that to look up `EOResourceID.STATUS_LABEL_ITEM_DROP_OUT_OF_RANGE`.

Looks like `"System"` isn't in the EDFs so that's fine to leave hard-coded.

Reply:
Usage example from recent `ADMININTERACT` change here: https://github.com/ethanmoffat/EndlessClient/commit/5d02996856f973075188f78676c584667786a217#diff-9a0776e511801824a93b00f7f2f9b499cea6fed37165d926f01058bed1213246R24

Reply:
Sure, I'll do that. My bad.

Reply:
If anyone can check this on a Linux setup, it would be greatly appreciated.  Unable to verify or debug the Linux build issues right now.

Reply:
This might be a build agent issue, I'll look into it. Not sure why it's failing. Your changes shouldn't impact it.